_posts/2023-02-13-changelog-188.markdown

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #188"
+date: 2023-02-13 13:00:00 +0300
+---
+
+
+## New Features
+
+* Detect new compiler errors (by [@kuksag]):
```

Comment:
Should we add `kuksag` here as external contributor?

Reply:
Discussed and decided together with @vlad20012